batch/batch/server/server.py

```diff
@@ -252,6 +252,12 @@ def create_job():
         if batch_id not in batch_id_batch:
             abort(404, 'valid request: batch_id {} not found'.format(batch_id))
 
+    if len(pod_spec.containers) != 1:
+        abort(400, f'only one container allowed in pod_spec {pod_spec}')
+
+    if pod_spec.containers[0].name != 'default':
+        abort(400, f'container name must be "default" was {pod_spec.containres[0].name}')
```

Comment:
Change to: `container name must be "default"; found "{pod_spec.containres[0].name}"`